File: Services/CoralTime.Services.API/CoralTime.DAL/Models/JiraSetting.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoralTime.DAL.Models.Member;

namespace CoralTime.DAL.Models
{
    public class JiraSetting : LogChanges.LogChanges
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public string Id { get; set; }

        public string SettingName { get; set; }

        public string UserEmail { get; set; }

        public string Domain { get; set; }

        public string ApiToken { get; set; }

        public int MemberId { get; set; }

        [ForeignKey("MemberId")]
        public virtual Member.Member Member { get; set; }
    }
}

File: Clients/coraltime.clients.web/src/app/models/jira-setting.ts
export class JiraSetting{
  id: string;
	settingName: string;
	userEmail: string;
	domain: string;
  apiToken: string;

	constructor(data = null) {
		if (data) {
			this.id = data.id;
			this.settingName = data.settingName;
			this.userEmail = data.userEmail;
			this.domain = data.domain;
      this.apiToken = data.apiToken;
		}
	}
}
